### ISSUE TITLE
Can this code train on VOC dataset? such as VOC2007?

### ISSUE THREAD

**Issue by jjprincess** [mahyarnajibi/SNIPER#73]
Has anybody train this code on pascal dataset? My dataset was made in pascal dataset , so I want to know if this code can train on pascal dataset or not.And how to do it.
Hope some guidance, thank you so much!

**Comment by mahyarnajibi** [comment]
Yes, the master branch now updated and can be trained and tested on the PASCAL VOC dataset. The result on VOC is reported in the readme and the models and negative proposals are released. Please pull the master and run the download scripts again.
